pages/content/amp-dev/documentation/courses/beginning-course/thinking-in-components@pt_BR.md

```diff
@@ -0,0 +1,265 @@
+
+Adicione o atributo `layout` à nossa imagem. Se feito corretamente, será semelhante a este:
+
+[sourcecode:html]
+{% raw %}<amp-img src="https://cdn.glitch.com/d7f46a57-0ca4-4cca-ab0f-69068dec6631%2Fricotta-racer.jpg?1540228217746" layout="responsive" width="640" height="480"></amp-img>
+{% endraw %}[/sourcecode]
+
+Depois de fazer a alteração, dê uma olhada em sua página. A imagem tem a proporção correta e preenche a largura da tela de maneira responsiva. Problema resolvido.
+
+{{ image('/static/img/courses/beginner/image26.png', 311, 550,  align='center third', caption='Imagem de bicicleta com proporção correta') }}
+
+Existem outros tipos de layout além do responsivo (pelo menos 8 no total, na verdade).
+Por exemplo, o layout `fixed` indica que o componente nunca deve ser redimensionado da altura e largura atribuídas a ele. O layout `intrinsic` é semelhante ao layout `responsive`, exceto que tem o conceito de altura e largura intrínsecas que não podem ser excedidas pelo componente. Alguns layouts só podem ser aplicados a determinados componentes. A documentação de cada componente especificará quais layouts são válidos para aquele componente. Você pode ler sobre o resto dos tipos de layout [aqui](../../../documentation/guides-and-tutorials/learn/amp-html-layout/layouts_demonstrated.html).
+
+Se você deseja se tornar um desenvolvedor de AMP de sucesso, aprender a usar o sistema de layout é fundamental. Todos os layouts fornecidos pelo AMP podem ser implementados usando CSS simples, mas geralmente podem ser complicados ou ter casos extremos complicados que exigem conhecimento profundo para serem solucionados. O AMP simplifica o processo e expõe muitas dessas opções de layout para serem usadas em qualquer elemento da página AMP. Confira o
+[documentação oficial](../../../documentation/guides-and-tutorials/learn/amp-html-layout/index.md) para obter mais informações sobre o sistema de layout.
+
+[tip type="tip"]
+**Dica**: Tente selecionar diferentes dispositivos móveis no menu suspenso (veja a captura de tela abaixo) para ver como a imagem se adapta a diferentes tamanhos de tela. É uma boa prática testar seu site em tamanhos de tela diferentes. Navegadores em dispositivos móveis reais podem se comportar de maneira diferente, então, quando possível, teste sua página da web em dispositivos reais também.
+
+{{ image('/static/img/courses/beginner/image4.png', 297, 380,  align='center half', caption='Lista suspensa de dispositivos no Chrome') }}
+[/tip]
+
+## Exercício 4: Incorporação de Vídeo
+
+A seguir, vamos incorporar um vídeo do YouTube em nosso documento. Nossa equipe de marketing da Chico’s Bikes lançou [este](https://www.youtube.com/watch?v=BlpMQ7fMCzA) vídeo de uma de nossas bicicletas queijos sendo construídas.
+
+Use a documentação do [`<amp-youtube>`](../../../documentation/components/reference/amp-youtube.md) para incorporar este vídeo do YouTube no componente `<amp-img>` com as seguintes configurações:
```

Comment:
is possible to add a locale path in those links? 

Such as:
```
[apêndice]({{g.doc('/content/amp-dev/documentation/courses/beginning-course/appendix.md', locale=doc.locale).url.path}}).
```

Reply:
Done!